src/frontend/contexts/PersistedStores/DraftObservationStore.ts

```diff
@@ -43,13 +46,17 @@ export function convertPosition(
   };
 }
 
-export function createDraftObservationStore({persist}: {persist: boolean}) {
+export function createDraftObservationStore(opts: CreateStoreOpts) {
   let nextAttachmentId = 0;
   let instance: StoreApi<DraftState>;
 
-  if (persist) {
+  if (opts.persist) {
     instance = createStore(
-      createPersistedState(createEmptyStoreState, {name: '@MapeoDraft'}),
+      createPersistedState(createEmptyStoreState, {
+        name: '@MapeoDraft',
+        storage: createJSONStorage(() => opts.storage),
+        version: 0,
+      }),
```

Comment:
this isn't being used by anything in the app yet, but if it were, it would've been not working properly 😅